main.go

```diff
@@ -0,0 +1,13 @@
+package main
+
+import (
+	"github.com/yelyzaveta-mykhalik/brainfuck/brainfuck"
+)
+
+func main() {
+
+	const helloWorld = "++++++++++[>+++++++>++++++++++>+++>+<<<<-]>++.>+.+++++++..+++.>++.<<+++++++++++++++.>.+++.------.--------.>+.>."
+
+	brainfuck.Execute(helloWorld)
```

Comment:
if i run this, I'm getting: 
```

goroutine 1 [running]:
github.com/yelyzaveta-mykhalik/brainfuck/brainfuck.Execute({0x10ba903, 0x6f})
        /Users/vitaliishevchenko/Users/vitaliishevchenko/teamdev_students/brainfuck/brainfuck/executor.go:25 +0x5ac
main.main()
        /Users/vitaliishevchenko/Users/vitaliishevchenko/teamdev_students/brainfuck/main.go:11 +0x25

Process finished with the exit code 2

```
pls fix that.

Reply:
fixed